build_all.bat

```diff
@@ -0,0 +1,44 @@
+echo "build from source on windows"
+
+rem requires `gcc` in your PATH
+rem one way is via calling finish.exe in command prompt after installing via
+rem https://nim-lang.org/install_windows.html
+rem note: this won't work if it's in a networked drive
+rem eg if using parallels on OSX, don't use a parallels shared folder, use C:\
+
+rem better to be verbose, for debugging purposes
+@echo on
+
+echo "checks that gcc in PATH:"
+gcc -v
+
+IF EXIST csources\nul (

```

Comment:
Why is there a null escape here?

Reply:
fixed (that depended on the stackoverflow answers)